rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -260,6 +260,7 @@ type Payout: <"Represents the set of future cashflow methodologies in the form o
         if optionPayout exists
         then optionPayout -> priceQuantity exists
                 or optionPayout -> underlier -> Product ->> economicTerms -> payout -> interestRatePayout count = 2
+                or optionPayout -> underlier -> Observable -> Asset -> Cash exists
```

Comment:
This last `or` (which was included in CDM 5 to cover the foreign exchange case) becomes unnecessary in CDM 6. Post the asset refactoring work, the quantity should be populated for foreign exchange options.

Reply:
@lolabeis - Changes made to the PR